pr-data.csv

```diff
@@ -3943,10 +3943,10 @@ https://github.com/jenkinsci/remoting,abf0455a68ad6c52a57e912bb89d51f883f77542,.
 https://github.com/jfree/jfreechart,520a4be69ba932061ab9b89f328842caf152e1d9,.,org.jfree.data.time.DayTest.testParseDay,OD-Vic,InspiredAFix,https://github.com/jfree/jfreechart/pull/103,
 https://github.com/jhipster/jhipster-registry,00db36611da5fc7aaf9d5372aa90f2465d80c0c4,.,io.github.jhipster.registry.web.rest.AccountResourceTest.testGetUnknownAccount,OD-Vic,DeveloperFixed,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/24
 https://github.com/jhipster/jhipster-registry,00db36611da5fc7aaf9d5372aa90f2465d80c0c4,.,io.github.jhipster.registry.web.rest.LogsResourceTest.changeLevelTest,ID,Deleted,,
-https://github.com/jitsi/jicofo,2cae36e1e6781b438e52156d82cc32f8dea021b9,.,org.jitsi.jicofo.auth.ShibbolethAuthenticationAuthorityTest.testShibbolethAuthenticationModule,OD,,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/83
-https://github.com/jitsi/jicofo,2cae36e1e6781b438e52156d82cc32f8dea021b9,.,org.jitsi.jicofo.auth.XMPPAuthenticationAuthorityTest.testXmppDomainAuthentication,OD,,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/83
-https://github.com/jitsi/jicofo,2cae36e1e6781b438e52156d82cc32f8dea021b9,.,org.jitsi.jicofo.RolesTest.testPassModeratorRole,OD,,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/83
-https://github.com/jitsi/jicofo,2cae36e1e6781b438e52156d82cc32f8dea021b9,.,org.jitsi.jicofo.XmppTest.testAllocateConference,OD,,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/83
+https://github.com/jitsi/jicofo,2cae36e1e6781b438e52156d82cc32f8dea021b9,.,org.jitsi.jicofo.auth.ShibbolethAuthenticationAuthorityTest.testShibbolethAuthenticationModule,OD,Deleted,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/83;https://github.com/jitsi/jicofo/commit/adaedbfd013e14462f53f7ffe616a3a186a26d51
+https://github.com/jitsi/jicofo,2cae36e1e6781b438e52156d82cc32f8dea021b9,.,org.jitsi.jicofo.auth.XMPPAuthenticationAuthorityTest.testXmppDomainAuthentication,OD,MovedOrRenamed,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/83;https://github.com/jitsi/jicofo/commit/305b5125537f75db0c8f26bae80e42871a943ac6
```

Comment:
`XMPPAuthenticationAuthorityTest.java` was deleted in `305b5125537f75db0c8f26bae80e42871a943ac6`. Please carefully double check `Deleted` vs. `MovedOrRenamed`. You can post somewhere privately justification for each of the four changed lines.

Reply:
I have solved merge conflicts and add justifications on CampusWire. Kindly review the changes. Thank you for your attention.